cocos2d/core/CCScheduler.js

```diff
@@ -200,6 +200,7 @@ proto.update = function (dt) {
     if (this._elapsed === -1) {
         this._elapsed = 0;
         this._timesExecuted = 0;
+        this.update(dt);
```

Comment:
It is the designed behavior, the scheduled tasks will always start after the first time interval ends.

Reply:
> It is the designed behavior, the scheduled tasks will always start after the first time interval ends.

I coded this, and the task ran after 2 frames. So, the  behavior is correct?
![2020-06-18_164001](https://user-images.githubusercontent.com/17826106/84998778-dd024600-b182-11ea-9980-1fdaf78a8fc7.png)
![2020-06-18_163924](https://user-images.githubusercontent.com/17826106/84998850-f4d9ca00-b182-11ea-9415-126b8c763dc8.png)

Reply:
Yes, actually the second "Component updated" and the first "myFunc updated" is printed in the same frame. The scheduler execution is later than the component update. So the behavior in each frame is like this:

- Frame 0:
    - start -> myFunc scheduled
    - update
- Frame 1:
    - update
    - myFunc call
- repeat Frame 1

Reply:
Okay, I got it.
I misunderstood  the log. Lol.
Thanks for your patience. :)

Reply:
So please revert this line, and we can merge your pr

Reply:
I got the 403 problem. Now it reverted.